app/views/members/home/index.html.haml

```diff
@@ -1,7 +1,47 @@
 - content_for(:page, 'home')
 
 .row
-  .col-lg-6.col-12
+  .col-md-6
+    .card
+      .card-header
+        %i.fas.fa-fw.fa-stream
+        = I18n.t("admin.navigation.messages")
```

Comment:
Kind of unnecessary

Reply:
Idk, I think it looks kinda empty without it.